cloudy_launcher.py

```diff
@@ -0,0 +1,110 @@
+'''
+Simulates the launcher for CloudyGame
+
+Starts a loop that does: 
+1. get user input:
+       join <username>
+       quit <username>
+       exit
+2. send join/quit command to CloudyPanel
+3a. receive controllerId if success.
+    start thin client with given controllerId.
+3b. receive error message if fail.
+
+'''
+
+import socket
+import errno
+import os
+import thin_client
+
+
+JOIN = "join"
+QUIT = "quit"
+EXIT = "exit"
+COMMANDS = "Commands: \n" + \
+           "connect and join user -- join <username> \n" + \
+           "connect and quit user -- quit <username> \n" + \
+           "exit cloudy_launcher  -- exit \n"
+
+
+if __name__ == "__main__":
+
+    ## to connect with CloudyPanel
+    TCP_IP = "127.0.0.1"
+    TCP_PORT = 55550
+    BUFFER_SIZE = 1024
+
+    print(COMMANDS)
+    
+    while (True):
+
+        playerInput = input("Enter command: ")
+        print(playerInput)
+
+        if (playerInput == EXIT):
+            break
+
+        elif ((len(playerInput.split()) == 0) | (len(playerInput.split()) != 2)):
+            print("Invalid command.\n")
+            print(COMMANDS)
+
+        elif (playerInput.split()[0] == JOIN):
```

Comment:
refactor all of these independent `split()` into `inputParams = playerInput.split()` as a  variable